README.rst

```diff
@@ -13,42 +13,98 @@ Installation
 
 Fontmake requires Python 2.7, 3.5 or later.
 
-Releases are available on `PyPI`_ and can be installed with `pip`_.
+There are three ways of getting fontmake.
+
+Installing from PyPI
+--------------------
+
+The standard route recommended for most people. Releases are available on
+`PyPI`_ and can be installed with `pip`_. In these instructions, we install
+fontmake in a virtual environment to avoid software clashes and permission
+problems. Open a terminal and follow along:
 
 .. code:: bash
 
+    # This assumes you have Python installed. If you are unsure, get Python 3
+    # from www.python.org.
+
+    # First, go to your project folder.
+
+    # Then, create a new virtual environment. You only need to do this the first
+    # time or if you deleted the old one.
+    # If you are using Python 2:
+    python -m virtualenv venv
+    # Otherwise, if you are using Python 3 on Mac or Linux:
+    python3 -m venv venv
+    # Otherwise, if you are using Python 3 on Windows:
+    py -3 -m venv venv
+
+    # Note that if you use Git to manage your project, you should add the line
+    # "venv" to the text file ".gitignore". Create the file in the top directory
+    # of your project if it doesn't exist. This will prevent accidentally
+    # committing the entire venv to Git, which is most probably not what you want.
+
+    # Activate the virtual environment. You need to do this every time you open
+    # a new terminal.
+    # On Mac or Linux, with the standard Bash shell:
+    . venv/bin/activate
+    # On Windows, with a cmd.exe terminal
+    .\venv\Scripts\activate.bat
+    # On Windows, in Powershell:
+    .\venv\Scripts\Activate.ps1
+
+    # Install fontmake.
     pip install fontmake
 
 Use the ``-U``, ``--upgrade`` option to update fontmake and its dependencies
 to the newest available release:
 
 .. code:: bash
 
-    pip install -U fontmake
+    # Activate virtual environment as above.
+
+    pip install --upgrade fontmake
+
+Using the stand-alone application
+---------------------------------
 
-Alternatively, you can download the git repository and install from source:
+If you have Python 3.6 or later installed, you can get a zip file with a
+stand-alone fontmake application from the [GitHub release page]_.
+
+1. Pick one for your platform and Python version, a "32" in the name needs a
+   32-bit Python installation, a "64" a 64-bit one. If you are unsure, better
+   follow the PyPI route above instead.
+2. Unzip it to where you want it, e.g. your project folder.
+3. Open a terminal and go to that folder.
+4. On Windows, run ``python fontmake.pyz``, on other platforms run
+   ``python3 fontmake.pyz``. If you see a message telling you about the options
```

Comment:
actually this is incorrect. The way you run the shiv bundled app is simply with its name (provided it is in the `PATH`), no need to call it with a specific python interpreter. On Windows, the `py` launcher will read the shebang and call the pyz app with the right interpreter. On Unix, bash or other shell will do the right thing (as long as `python3.6` or `python3.7` -- depending on which `fontmake.pyz` was downloaded -- are in the `PATH`).

Reply:
On Windows, the `.pyz` extension is normally registered to be executed by the `py` python launcher which is installed by default with Python.org installers. If that is not the case for whatever reason, then simpy running `fontmake.pyz` may not work out of the box, and you may need to either associated `.pyz` extesion with the python launcher, or call it as `py fontmake.pyz`